codejam-canvas/script.js

```diff
@@ -0,0 +1,6268 @@
+      ],
+      [
+        216,
+        49,
+        33,
+        172
+      ],
+      [
+        165,
+        49,
+        39,
+        205
+      ],
+      [
+        117,
+        49,
+        43,
+        255
+      ],
+      [
+        128,
+        49,
+        42,
+        255
+      ],
+      [
+        134,
+        49,
+        41,
+        255
+      ],
+      [
+        155,
+        49,
+        39,
+        255
+      ],
+      [
+        179,
+        49,
+        36,
+        255
+      ],
+      [
+        174,
+        49,
+        37,
+        255
+      ],
+      [
+        186,
+        49,
+        36,
+        255
+      ],
+      [
+        174,
+        49,
+        37,
+        255
+      ],
+      [
+        153,
+        49,
+        39,
+        255
+      ],
+      [
+        80,
+        49,
+        47,
+        255
+      ],
+      [
+        13,
+        49,
+        55,
+        255
+      ],
+      [
+        21,
+        49,
+        54,
+        255
+      ],
+      [
+        27,
+        49,
+        53,
+        255
+      ],
+      [
+        30,
+        49,
+        53,
+        255
+      ],
+      [
+        20,
+        49,
+        54,
+        255
+      ],
+      [
+        19,
+        49,
+        54,
+        255
+      ],
+      [
+        21,
+        49,
+        54,
+        255
+      ],
+      [
+        71,
+        49,
+        48,
+        255
+      ],
+      [
+        94,
+        49,
+        46,
+        255
+      ],
+      [
+        153,
+        49,
+        39,
+        255
+      ],
+      [
+        168,
+        49,
+        37,
+        255
+      ],
+      [
+        152,
+        49,
+        39,
+        255
+      ],
+      [
+        156,
+        49,
+        39,
+        255
+      ],
+      [
+        156,
+        49,
+        39,
+        255
+      ],
+      [
+        167,
+        49,
+        38,
+        255
+      ],
+      [
+        169,
+        49,
+        37,
+        255
+      ],
+      [
+        157,
+        49,
+        39,
+        254
+      ],
+      [
+        196,
+        49,
+        35,
+        181
+      ],
+      [
+        214,
+        48,
+        33,
+        69
+      ]
+    ],
+    [
+      [
+        207,
+        48,
+        34,
+        106
+      ],
+      [
+        215,
+        49,
+        33,
+        183
+      ],
+      [
+        172,
+        49,
+        39,
+        225
+      ],
+      [
+        130,
+        49,
+        41,
+        255
+      ],
+      [
+        120,
+        49,
+        43,
+        255
+      ],
+      [
+        142,
+        49,
+        40,
+        255
+      ],
+      [
+        157,
+        49,
+        39,
+        255
+      ],
+      [
+        153,
+        49,
+        39,
+        255
+      ],
+      [
+        152,
+        49,
+        39,
+        255
+      ],
+      [
+        159,
+        49,
+        38,
+        255
+      ],
+      [
+        154,
+     
```

Comment:
I'm not sure, but you can use `const` here

Reply:
edited